blog/content/post/2020-10-19-security-part-2.md

```diff
@@ -0,0 +1,127 @@
+
+- Injection
+
+Code injection occurs when an attacker sends invalid data to a web application with the intent to do something the application was not designed to do.
+
+- Broken Authentication
+
+Broken authentication vulnerability could allow an attacker to use manual or automated methods to take control of any account on the system or worse, to gain complete control of the system.
+
+- Sensitive Data Exposure
+
+Exposure of sensitive data is one of the most common vulnerabilities on the OWASP list. It involves the disclosure of data that should be protected.
+
+- XML External Entities (XXE)
+
+An XML External Entity attack is a type of attack against an application that parses XML input. This attack occurs when XML input containing a reference to an external entity is processed by a weakly configured XML parser.
+
+- Broken Access control
+
+Broken access control means limiting which sections or pages visitors can reach, depending on their needs.
+
+- Security misconfigurations
+
+The problem may be related to, for example, keeping the default CMS configurations. There are settings you can adjust to control comments, users, and visibility of user information. File permissions are another example of a default that can be strengthened.
+
+- Cross Site Scripting (XSS)
+
+XSS is a widespread vulnerability that affects many web applications. XSS attacks involve injecting malicious client-side scripts into a website and using that website as a propagation method.
+
+- Insecure Deserialization
+
+An example of this type of security risk is a super cookie that contains serialized information about a logged in user. This cookie defines the role of the user and could be a vulnerability that may endanger the entire web application.
+
+- Using Components with known vulnerabilities
+
+Today, even simple websites like personal blogs have many dependencies.
+We can all agree that not updating every software on the backend and front end of the website will no doubt introduce serious security threats sooner rather than later.
+
+- Insufficient logging and monitoring
+
+The importance of website security cannot be underestimated. There are ways to monitor your site regularly so that you can take immediate action when something happens. Lack of an efficient login and monitoring process can increase the damage caused by a website hack.
+
+[Official website of the organization](https://owasp.org/www-project-top-ten/)
+
+The list is periodically published and importantly, consists only 10 points. This makes it easy to analyze and understand, making it a good start to taking a serious approach to security in your organization.
+
+OWASP list should be included in every software development project.
+It is the best set of security requirements and best practices, and most importantly, it works!
+Following and implementing the OWASP recommendations has only positive consequences and teaches you to apply the best available practices.
+
```

Comment:
It is ok to describe OWASP, but we need in this  blogpost some concrete examples that will say: "hey, we are the good guys, here in 3mdeb we do take our responsibility. Look at our contirbution, or project or any_thing_that_will_come_to_your_mind. Seek for an inspiiration in blog, or  here: https://opensource.3mdeb.com/
or here https://3mdeb.com/events/ or elsewhere. Prove, that we, as 3mdeb do take care about.